docs-gb/README.md

```diff
@@ -1,63 +1,48 @@
-* Dynamically extended models with pipelines with a data-centric perspective backed by Kafka.
-* Explain individual models and pipelines with state of the art explanation techniques.
-* Deploy drift and outlier detectors alongside models.
-* Kubernetes Service mesh agnostic - use the service mesh of your choice.
+Seldon Core 2 orchestrates and scales machine learning components running as production-grade microservices. These components can be deployed locally or in enterprise-scale kubernetes clusters. The components of your ML system - such as models, processing steps, custom logic, or monitoring methods - are deployed as **Models**, leveraging serving solutions compatible with Core 2 such as MLServer, Alibi, LLM Module, or Triton Inference Server. These serving solutions package the required dependencies and standardize inference using the Open Inference Protocol. This ensures that, regardless of your model types and use-cases, all request and responses follow a unified format. Once models are deployed, they can process REST or gRPC requests for real-time inference.
 
+## Complex Applications & Orchestration
 
-## Core features and comparison to Seldon Core V1 APIs
+Machine learning applications are increasingly complex. They’ve evolved from individual models deployed as services, to complex applications that can consist of multiple models, processing steps, custom logic, and asynchronous monitoring components. With Core you can build  Pipelines that connect any of these components to make data-centric applications. Core 2 then handles the orchestration and scaling of the underlying components of such an application, and exposes the data streamed through the application in real time using Kafka. 
 
-Our V2 APIs separate out core tasks into separate resources allowing users to get started fast
-with deploying a Model and the progressing to more complex Pipelines, Explanations and Experiments.
+{% hint style="info" %}
+Data-centricity is an approach that places the management, integrity, and flow of data at the core of the machine learning deployment framework. 
+{% endhint %}
 
-![intro](images/intro.png)
+This approach to MLOps, influenced by our position paper [Desiderata for next generation of ML model serving](https://arxiv.org/abs/2210.14665), enables real-time observability, insight, and control on the behavior, and performance of your ML systems.  
 
-## Multi-model serving
 
-Seldon transparently will provision your model onto the correct inference server.
+![Data-centric pipeline](images/pipeline-intro.png)
 
-![mms1](images/multimodel1.png)
+Lastly, Core 2 provides Experiments as part of its orchestration capabilities, enabling users to implement routing logic like A/B tests or Canary deployments to models or pipelines in production. After experiments are run, you can promote new models or pipelines, or launch new experiments, allowing you to continuously improve the performance of your ML products.
```

Comment:
```suggestion
Lastly, Seldon Core 2 provides Experiments as part of its orchestration capabilities, to implement routing logic such as  A/B tests or Canary deployments to models or pipelines in production. After experiments are run, you can promote new models or pipelines, or launch new experiments, so that you can continuously improve the performance of your ML products.
```